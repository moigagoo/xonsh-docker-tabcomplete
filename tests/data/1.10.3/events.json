{
        "arguments": [
                "-f", "--filter=[]",
                "--help",
                "--since",
                "--until"
        ],
        "commands": []
}
