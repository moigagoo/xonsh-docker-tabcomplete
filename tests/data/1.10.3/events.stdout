
Usage:	docker events [OPTIONS]

Get real time events from the server

  -f, --filter=[]    Filter output based on conditions provided
  --help             Print usage
  --since            Show all events created since timestamp
  --until            Stream events until this timestamp
